index.html

```diff
@@ -67,7 +67,11 @@ <h1 class="h1">
                 </div>
             </section>
             <section id="work-container">
```

Comment:
All of your work section cards are breaking all over the place. There is no resemblance with the Figma design of your choosing. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178109573-9ec8b53f-a9e0-487e-91a9-4bd3436fab29.png) ![image](https://user-images.githubusercontent.com/33268927/178109600-1ea84704-5c9a-4036-bace-764d01370429.png) | ![image](https://user-images.githubusercontent.com/33268927/178109621-cc6eb385-064c-415c-beea-58f0cd4a2e7e.png) |